### PR TITLE
Update ZE RNs

### PR DESCRIPTION
**Pull request by IgorCATech**
Signed-off-by: Igor Kazmyr <igor.kazmyr@broadcom.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
This PR will update the Zowe Explorer release note in the general releases note document.

:heart:Thank you!

### CODE REVIEW

**Review by nannanli — APPROVED** [review]
LGTM

File: docs/getting-started/summaryofchanges.md [hunk 1]
```diff
@@ -133,13 +133,7 @@ The following bug was fixed in the **FTP Plug-in**:
...
 #### Zowe Explorer
 
-- Fixed the issue that required the vscode module to be imported in the API package [#1318](https://github.com/zowe/vscode-extension-for-zowe/pull/1318) 
-- Fixed the issue that prevented the `issue STOP command` function from executing correctly [#1304](https://github.com/zowe/vscode-extension-for-zowe/pull/1304) 
-- Fixed the issue that caused the Add Profile icon to disappear [#1307](https://github.com/zowe/vscode-extension-for-zowe/pull/1307)
-- Fixed the vulnerability in NPM Audit [#1309](https://github.com/zowe/vscode-extension-for-zowe/pull/1309)
-- Fixed the issue that doubled the occurrence of the port prompt [#1298](https://github.com/zowe/vscode-extension-for-zowe/pull/1298)
-- Fixed the issue that triggered the `Delete Job` command even outside Zowe Explorer views [#1310](https://github.com/zowe/vscode-extension-for-zowe/pull/1310)
-- Fixed the trailing slash issue that caused issues with USS search [#1313](https://github.com/zowe/vscode-extension-for-zowe/pull/1313)
+- Fixed the error message that popped up when accessing profiles from Favorites. [#1344](https://github.com/zowe/vscode-extension-for-zowe/pull/1344)
```
Comment (crawr) [inline docs/getting-started/summaryofchanges.md:137]: We also fixed the command `Allocate Like` which was broken in the previous version. Related PR: https://github.com/zowe/vscode-extension-for-zowe/pull/1322

**Review by crawr — COMMENTED** [review]
Everything looks great. I just commented on 1 missing bug fix that will go in with 1.16.0. Thanks @IgorCATech

File: docs/getting-started/summaryofchanges.md (hunk 1 above)
Comment (IgorCATech) [reply docs/getting-started/summaryofchanges.md:137]: let me add that rq

**Review by crawr — APPROVED** [review]
LGTM :)

**Pull request merged**